### PR TITLE
New moj-official preproduction account

### PR DESCRIPTION
**Pull request by SteveLinden**
New account for moj-offcial preproduction. There is already a "MOJ Official (Production)" account so this one should not impact that one

### CODE REVIEW

**Review by markgov — DISMISSED** [review]
LGTM

**Review by davidkelliott — CHANGES_REQUESTED** [review]
Fix github slug

File: environments/moj-official.json [hunk 1]
```diff
@@ -0,0 +1,20 @@
+{
+  "account-type": "member",
```
Comment (davidkelliott) [inline environments/moj-official.json:2]: This should be member unrestricted

**Review by markgov — APPROVED** [review]
LGMT

**Pull request merged**